reflex-dhtmlx/src/Reflex/Dom/DHTMLX/Date.hs

```diff
@@ -75,12 +76,14 @@ data DatePickerConfig t = DatePickerConfig
     , _datePickerConfig_weekStart     :: WeekDay
     , _datePickerConfig_attributes    :: Dynamic t (Map Text Text)
     , _datePickerConfig_visibleOnLoad :: Bool
+    -- | a function from the date update event to an event that hides the date picker. Defaulted to id.
+    , _datePickerConfig_hideRule      :: Event t () -> Event t ()
```

Comment:
Can you provide more info on this? Is the idea here that we provide the user with an `Event t ()` and let them decide when to close by returning a new `Event t ()`?

Reply:
My understanding was "given the 'date updated' event, decide when to hide the date picker" - which would allow delaying some amount of time or doing `const never`.

Reply:
Yes - you can also implement more complicated potentially-stateful logic outside of the picker here and use "const myFancyCloseEvent" if you really want to, but I didn't want to add the complication of an m or other events than the default one to the interface.